### PR TITLE
JWT auth implemented, Swagger reconfigured to work with JWT, Readme a…

### PR DESCRIPTION
**Pull request by semyonich**
…dded

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+currency exchange rate, which is obtained from side [API](https://api.exchangerate.host).
+You can test this app through Swagger UI, 
+which is available at http://localhost:8080/swagger-ui/. 
+All input data is being validated.
+These features are realized with:
+- MySQL (DB layer)
+- Java (Service layer)
+- Spring Boot 
+- Spring Security (Authentication/Authorization) with JWT
+- Swagger  <br>
```
Comment (boroda4436) [inline README.md]:
```suggestion
- Swagger
```

File: src/main/java/com/pet/bankservice/controller/AuthenticationController.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@AllArgsConstructor
+public class AuthenticationController {
+    private final AuthenticationManager manager;
```
Comment (boroda4436) [inline src/main/java/com/pet/bankservice/controller/AuthenticationController.java]:
```suggestion
    private final AuthenticationManager authManager;
```

File: src/main/java/com/pet/bankservice/security/jwt/JwtUser.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+import java.util.Collection;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+@AllArgsConstructor
+@Data
+public class JwtUser implements UserDetails {
```
Comment (boroda4436) [inline src/main/java/com/pet/bankservice/security/jwt/JwtUser.java:15]:
```suggestion
public class User implements UserDetails {
```
I guess there is no need to call the user as a JwtUser

File: src/main/java/com/pet/bankservice/security/jwt/JwtUserMapper.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package com.pet.bankservice.security.jwt;
+
+import com.pet.bankservice.entity.User;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+public final class JwtUserMapper {
+    public static JwtUser create(User user) {
```
Comment (boroda4436) [inline src/main/java/com/pet/bankservice/security/jwt/JwtUserMapper.java]: Should we have two separate classes? Can we have only one `User` class?

File: .travis.yml [hunk 5]
```diff
@@ -1,4 +1,6 @@
 language: java
+services:
+  - mysql
```
Comment (boroda4436) [inline .travis.yml:3]: Дуже круто! :muscle:

**Review by boroda4436 — CHANGES_REQUESTED** [review]
Please see the comment about two user classes.

File: README.md (hunk 1 above)
Comment (semyonich) [reply README.md]: done

File: src/main/java/com/pet/bankservice/controller/AuthenticationController.java (hunk 2 above)
Comment (semyonich) [reply src/main/java/com/pet/bankservice/controller/AuthenticationController.java]: done

File: src/main/java/com/pet/bankservice/security/jwt/JwtUser.java (hunk 3 above)
Comment (semyonich) [reply src/main/java/com/pet/bankservice/security/jwt/JwtUser.java:15]: I don't like idea to have 2 classes with the same name, and also I don't want to change my User model so I would like to leave this as is

File: src/main/java/com/pet/bankservice/security/jwt/JwtUserMapper.java (hunk 4 above)
Comment (semyonich) [reply src/main/java/com/pet/bankservice/security/jwt/JwtUserMapper.java]: done

File: .travis.yml (hunk 5 above)
Comment (semyonich) [reply .travis.yml:3]: without it travis fails

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**